bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncTable.java

```diff
@@ -229,8 +226,8 @@ public CheckAndMutateBuilder timeRange(TimeRange timeRange) {
   /** {@inheritDoc} */
   @Override
   public CompletableFuture<Result> get(Get get) {
-    return toCompletableFuture(clientWrapper.readFlatRowsAsync(hbaseAdapter.adapt(get)))
-        .thenApply(BigtableAsyncTable::toResult);
+    return toCompletableFuture(clientWrapper.readRowsAsync(hbaseAdapter.adapt(get)))
```

Comment:
In another PR, we should introduce a point read method to DataClientWrapper and use that for point reads. This will allow us to tune deadlines separately for scans and point reads